src/index.tsx

```diff
@@ -87,6 +94,22 @@ export async function trackEvent(
   return await PiwikProSdk.trackEvent(category, action, { name, value });
 }
 
+/**
+ * Set a custom dimension.
```

Comment:
Might be good to also provide a way to unset a custom dimension, especially with the `visit` scope.